src/actions.js

```diff
@@ -190,10 +191,37 @@ export const setUniqueDeviceId = (uniqueId) => ({
   payload: uniqueId,
 });
 
+/**
+ * Action to set the screen unlocked.
+ * @returns action
+ */
 export const unlockScreen = () => ({ type: types.SET_LOCK_SCREEN, payload: false });
 
+/**
+ * Check if the action is about to set screen to unlocked state.
+ * @param {{ type: string, payload: boolean }} action
+ * @returns {boolean} true if unlocked and false otherwise.
+ */
+export const isUnlockScreen = (action) => action.type === types.SET_LOCK_SCREEN && action.payload === false;
```

Comment:
This is not an action, it's just a helper method, I think we should move it to `sagas/pushNotification.js` or to `sagas/helpers.js`

Reply:
Also, just a suggestion, maybe rename it to `willUnlock`, what do you think?

Reply:
I agree with the helper, but not the rename. `willUnlock` denotes some action in the future, but the action was already triggered, therefore it is near the present time, or strictly already in the past.